### DISCUSSION TITLE
Something is wrong with check 50? Expected and not outputs are literally the same. Lab1 population

### DISCUSSION THREAD

**Discussion by AgniprabhaGanguly** [mareksuscak/cs50#16] [Show and tell]
![IMG_20230627_010939.jpg](https://github.com/mareksuscak/cs50/assets/135814778/96765978-80ef-4e9d-8dbf-aa756c461d6f)

**Comment by mareksuscak** [comment]
@AgniprabhaGanguly thank you for reporting. These solutions have not been updated since 2018 and there's a chance that the problem sets have changed since then.

**Reply by AgniprabhaGanguly** [reply]
I see

**Reply by Jtobyy** [reply]
Has your issue been resolved?

**Reply by shubhamyadav8901** [reply]
`Years : 2` is different from `Years: 2`. Check50 will check for the space as well. So delete the extra space and Check50 will work fine.
